example/client.py

```diff
@@ -1,66 +1,35 @@
-from flask import Flask
-from flask_opentracing import FlaskTracer
-import lightstep.tracer
-import opentracing
-import urllib2
+import requests
+import time
+from opentracing_instrumentation.client_hooks import install_all_patches
+from opentracing_instrumentation.request_context import get_current_span, span_in_context
+from jaeger_client import Config
+from opentracing.ext import tags
+from opentracing.propagation import Format
 
-app = Flask(__name__)
+from os import environ
 
-# one-time tracer initialization code
-ls_tracer = lightstep.tracer.init_tracer(group_name="example client", access_token="{your_lightstep_token}")
-# this tracer does not trace all requests, so the @tracer.trace() decorator must be used
-tracer = FlaskTracer(ls_tracer)
+JAEGER_HOST = environ.get('JAEGER_HOST')
 
-@app.route("/")
-def index():
-	'''
-	Index page, has no tracing.
-	'''
-	return "Index Page"
+WEBSERVER_HOST = environ.get('WEBSERVER_HOST')
 
+config = Config(config={'sampler': {'type': 'const', 'param': 1},
```

Comment:
this could use a comment explaining what's happening, i.e. "configuring and creating an instance of Jaeger Tracer with 100% sampling and debug logging of spans".